### PR TITLE
DOC-857 Update metadata descriptions in postgres_cdc

### PR DESCRIPTION
**Pull request by asimms41**
## Description

Resolves [DOC-857](https://redpandadata.atlassian.net/browse/DOC-857)
Review deadline: 12th December

## Page previews

[postgres_cdc](https://deploy-preview-126--redpanda-connect.netlify.app/redpanda-connect/components/inputs/postgres_cdc/#metadata)

## Checks

- [ ] New feature
- [x] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

[DOC-857]: https://redpandadata.atlassian.net/browse/DOC-857?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: modules/components/pages/inputs/postgres_cdc.adoc [hunk 1]
```diff
@@ -163,9 +163,9 @@ ALTER TABLE large_data REPLICA IDENTITY FULL;
 
 This input adds the following metadata fields to each message:
 
-- `mode`: Whether a message is part of the snapshot processing of existing data in a database (`snapshot`) or a data update streamed to Redpanda Connect (`streaming`).
 - `table`: The name of the database table from which the message originated.
-- `operation`: The type of database operation that generated the message, such as `INSERT`, `UPDATE`, `DELETE`, `BEGIN` and `COMMIT`. `BEGIN` and `COMMIT` operations are only included if the `include_transaction_markers` field is set to `true`.
+- `operation`: The type of database operation that generated the message, such as `READ`, `INSERT`, `UPDATE`, `DELETE`, `BEGIN` and `COMMIT`. `READ` operations occur when a snapshot of the database is processed. `BEGIN` and `COMMIT` operations are only included if the `include_transaction_markers` field is set to `true`.
+- `lsn`: The https://www.postgresql.org/docs/current/wal-internals.html[Log Sequence Number^] of each data update from the source PostgreSQL database.
```
Comment (rockwotj) [inline modules/components/pages/inputs/postgres_cdc.adoc:168]: Can we use https://www.postgresql.org/docs/current/datatype-pg-lsn.html as the documentation reference? Also can we mention that this is a string and that it's formatting in a way that it's string ordering results in the same order as documented in the postgres documentation?

The sort order details can be useful for some work on the Snowflake connector that I am doing

Comment (asimms41) [reply modules/components/pages/inputs/postgres_cdc.adoc]: Updated. @rockwotj, please check for accuracy.

**Review by JakeSCahill — APPROVED** [review]

File: modules/components/pages/inputs/postgres_cdc.adoc [hunk 2]
```diff
@@ -163,9 +163,9 @@ ALTER TABLE large_data REPLICA IDENTITY FULL;
 
 This input adds the following metadata fields to each message:
 
-- `mode`: Whether a message is part of the snapshot processing of existing data in a database (`snapshot`) or a data update streamed to Redpanda Connect (`streaming`).
 - `table`: The name of the database table from which the message originated.
-- `operation`: The type of database operation that generated the message, such as `INSERT`, `UPDATE`, `DELETE`, `BEGIN` and `COMMIT`. `BEGIN` and `COMMIT` operations are only included if the `include_transaction_markers` field is set to `true`.
+- `operation`: The type of database operation that generated the message, such as `READ`, `INSERT`, `UPDATE`, `DELETE`, `BEGIN` and `COMMIT`. `READ` operations occur when a snapshot of the database is processed. `BEGIN` and `COMMIT` operations are only included if the `include_transaction_markers` field is set to `true`.
```
Comment (rockwotj) [inline modules/components/pages/inputs/postgres_cdc.adoc:167]: These should be lowercase.

**Review by rockwotj — APPROVED** [review]

File: modules/components/pages/inputs/postgres_cdc.adoc (hunk 2 above)
Comment (asimms41) [reply modules/components/pages/inputs/postgres_cdc.adoc:167]: @rockwotj All the operations should be in lowercase?

Comment (rockwotj) [reply modules/components/pages/inputs/postgres_cdc.adoc:167]: yes

**Pull request merged**